plugins/tekton/src/components/pipeline-topology/PipelineVisualization.tsx

```diff
@@ -147,12 +192,31 @@ export const PipelineVisualization = ({
         </div>
       ) : (
         <div
-          data-testid="pipeline-visualization"
+          data-testid={
```

Comment:
can we avoid this? the component should be declarative and ideally not good to change test id conditionally